packages/frontend/app/routes/assets.$assetId.deposit-liquidity.tsx

```diff
@@ -4,6 +4,7 @@ import { v4 } from 'uuid'
 import { LiquidityDialog } from '~/components/LiquidityDialog'
```

Comment:
This file, and other files in `/app/routes/*.tsx` all generally involve modifying the Remix loader and/or action to retrieve the API auth token from the session and passing it to whatever API call it's making.